src/components/AccountForm.vue

```diff
@@ -703,6 +703,19 @@ export default {
 	font-weight: bold;
 }
 
+:deep(.checkbox-radio-switch) {
+	border: 0 !important;
+}
+
+:deep(.input-field) {
+	margin-bottom: calc(var(--default-grid-baseline) * 3);
+	margin-top: var(--default-grid-baseline);
+}
+
+label {
+	margin-top: calc(var(--default-grid-baseline) * 2);
```

Comment:
as an alternative we could convert every HTML label+input into NcInput. It is even more compact

Reply:
Done